swf/src/avm2/read.rs

```diff
@@ -171,6 +171,20 @@ impl<'a> Reader<'a> {
             0x1c => Multiname::MultinameLA {
                 namespace_set: self.read_index()?,
             },
+            0x1d => {
+                let base_type = self.read_index()?;
+                let count = self.read_u30()?;
+                let mut parameters = Vec::new();
```

Comment:
You can do `Vec::with_capacity(count as usize)` to avoid unnecessary re-allocations.